pkgs/spark-wallet/generate.sh

```diff
@@ -1,3 +1,26 @@
 #!/usr/bin/env nix-shell
-#! nix-shell -i bash -p nodePackages.node2nix
-exec node2nix --nodejs-10 -i pkg.json -c composition.nix --no-copy-node-env --supplement-input supplement.json
+#! nix-shell -i bash -p nodePackages.node2nix gnupg wget jq moreutils
+set -euo pipefail
+
+TMPDIR="$(mktemp -d -p /tmp)"
+trap "rm -rf $TMPDIR" EXIT
+
+# Get/verify spark-wallet-npm.tgz
+version=$(cat version)
```

Comment:
```suggestion
version="0.2.14"

# Get/verify spark-wallet-npm.tgz
```
and remove file `version`. I think an extra file for just the version is unnecessary.

Reply:
Resolved in e1527c7